suite-common/bluetooth/tests/bluetoothSelectors.test.ts

```diff
@@ -0,0 +1,48 @@
+import { BluetoothDeviceState, prepareSelectAllDevices } from '../src';
+import { BluetoothDeviceCommon, BluetoothState } from '../src/bluetoothReducer';
+import { WithBluetoothState } from '../src/bluetoothSelectors';
+
+const initialState: BluetoothState<BluetoothDeviceCommon> = {
+    adapterStatus: 'unknown',
+    scanStatus: 'idle',
+    nearbyDevices: [] as BluetoothDeviceState<BluetoothDeviceCommon>[],
+    knownDevices: [] as BluetoothDeviceCommon[],
+};
+
+describe('bluetoothSelectors', () => {
+    it('selects knownDevices and nearbyDevices in one list fot the UI', () => {
+        const selectAllDevices = prepareSelectAllDevices<BluetoothDeviceCommon>();
+
+        const pairingNearbyDeviceA: BluetoothDeviceState<BluetoothDeviceCommon> = {
+            device: {
+                id: 'A',
+                data: [],
+                name: 'Trezor A',
+                lastUpdatedTimestamp: 1,
+            },
+            status: { type: 'pairing' },
+        };
+
+        const knownDeviceB: BluetoothDeviceCommon = {
+            id: 'B',
+            data: [],
+            name: 'Trezor B',
+            lastUpdatedTimestamp: 2,
+        };
+
+        const state: WithBluetoothState<BluetoothDeviceCommon> = {
+            bluetooth: {
+                ...initialState,
+                nearbyDevices: [pairingNearbyDeviceA],
```

Comment:
if the [comment in selectors file](https://github.com/trezor/trezor-suite/pull/17090#discussion_r1965335492) makes sense, then the test should be expanded to cover it:
```suggestion
                nearbyDevices: [pairingNearbyDeviceA, pairingNearbyDeviceB],
```